code/datums/components/material_container.dm

```diff
@@ -43,14 +45,17 @@
 			materials[id] = new mat_path()
 
 /datum/component/material_container/proc/OnExamine(mob/user)
-	for(var/I in materials)
-		var/datum/material/M = materials[I]
-		var/amt = amount(M.id)
-		if(amt)
-			to_chat(user, "<span class='notice'>It has [amt] units of [lowertext(M.name)] stored.</span>")
+	if(show_on_examine)
```

Comment:
Apparently showing materials always happens now, as this was never put in, despite there being arguments for it.